### PR TITLE
update zap usage in spec

### PR DESCRIPTION
**Pull request by ethanfrogers**

### CODE REVIEW

File: lifecycle/server.go [hunk 1]
```diff
@@ -9,7 +9,8 @@ import (
 	"time"
 
 	"github.com/skuid/spec"
-	"github.com/uber-go/zap"
+
```
Comment (micahhausler) [inline lifecycle/server.go]: Could you keep non-stdlib imports in the same block? Same for all files.

File: spec.go [hunk 2]
```diff
@@ -25,20 +25,41 @@ The Logger can be used like so:
...
-			// TODO: Write a stacktrace key formatter for zap
-		),
-	)
+var Logger *zap.Logger
+
+func Init(l *zap.Logger) {
+	Logger = l
+}
+
+func NewStandardLogger() (l *zap.Logger, err error) {
```
Comment (micahhausler) [inline spec.go]: Can you add a comment to exported functions?

File: spec.go [hunk 3]
```diff
@@ -25,20 +25,41 @@ The Logger can be used like so:
...
-		zap.NewJSONEncoder(
-			zap.RFC3339Formatter("timestamp"),
-			zap.MessageKey("message"),
-			zap.LevelString("level"),
-			// TODO: Write a stacktrace key formatter for zap
-		),
-	)
+var Logger *zap.Logger
+
+func Init(l *zap.Logger) {
```
Comment (micahhausler) [inline spec.go]: Can we cut this function out? Its unnecessary

**Comment by ethanfrogers** [comment]
@micahhausler Updated!

File: spec.go [hunk 4]
```diff
@@ -25,20 +25,38 @@ The Logger can be used like so:
 package spec
 
 import (
-	"github.com/uber-go/zap"
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
 )
 
 // Logger is a zap.Logger that is initialized with the proper field names and
```
Comment (micahhausler) [inline spec.go]: Since this is no longer true, could you also update the comment doc?

**Comment by ethanfrogers** [comment]
@micahhausler Lets get this merged at some point.

**Pull request merged**